src/main/kotlin/com/wire/apps/polls/services/StatsFormattingService.kt

```diff
@@ -115,14 +127,26 @@ private data class VotingOption(
     val option: String,
     val votingUsers: Int
 ) {
-    private companion object {
-        const val NOT_VOTE = "⚪"
-        const val VOTE = "🟢"
+    fun toString(max: Int): String {
+        val voteDisplay = VoteDisplay(votingUsers, max)
+
+        return "$voteDisplay $style$option$style ($votingUsers)"
+    }
+}
+
+object VotingCount {
```

Comment:
I don't get the meaning of this object, can't we move the constants and functionalities directly into VoteCount and remove this?
Having the logic in 2 objects with very similar name can be misleading